### PR TITLE
Some fixes from PHPStan l2

### PR DESCRIPTION
**Pull request by carusogabriel**
I found an interesting one for our CS with this PR:

```diff
- /* @var Class $variable */
+ /** @var Class $variable */
```

/cc @Majkl578 @kukulich

### CODE REVIEW

**Comment by Ocramius** [comment]
Yeah, phpstan set the standard for inline IDE hints there (before, `/*` and `*/` were to be used (see also https://netbeans.org/bugzilla/show_bug.cgi?id=267470)

**Comment by kukulich** [comment]
Hmm, there's already `SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration` for the format of inline PHPDoc but it looks it should also check `/*`.

**Comment by Majkl578** [comment]
I don't really think `/* ... */` was ever supposed to be a doc block. Correct is `/** ... */ `: https://docs.phpdoc.org/guides/docblocks.html#anatomy-of-a-docblock

**Review by Majkl578 — DISMISSED** [review]
Can we enable L2 on Travis too now?

File: composer.lock [hunk 1]
```diff
@@ -294,12 +294,12 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/doctrine/dbal.git",
-                "reference": "3c0bad37f41182de2b49ce5c4f8935ae7d808c81"
+                "reference": "e6e2eeab79a91c15be6431dcf334ea082b0f657b"
```
[the remark this answers is not in the document]
Comment (carusogabriel) [reply composer.lock]: Should I revert it?

Hate have `.lock` versioned, always do that :sweat_smile:

[the remark this answers is not in the document]
Comment (Ocramius) [reply composer.lock]: Yes please: `.lock` should not be randomly changed, or else we don't know why build-related deps crash our builds.

[the remark this answers is not in the document]
Comment (Majkl578) [reply composer.lock]: Probably, yes.
You can use the following to ignore any changes to `composer.lock`:
```bash
$ git update-index --assume-unchanged composer.lock
```
Then even if you run `composer update` and update dependencies, git will ignore that. ;)

[the remark this answers is not in the document]
Comment (carusogabriel) [reply composer.lock]: @Ocramius @Majkl578 I'm learning more and more about Git every single day with you, thanks <3

I have reverted these changes, should be RTM :)

**Review by Ocramius — APPROVED** [review]
:+1:

**Pull request merged**